public/endpoints.js

```diff
@@ -1,11 +1,9 @@
 window.endpoints = {
-  elasticsearch: 'http://test_domain.com',
-  results: 'http://test_domain.com',
-  graphql: 'http://test_domain.com',
-  prefix: 'test_prefix.',
-  run_index: 'vn.run-data.',
-  run_toc_index: 'vn.run-toc.',
-  result_index: 'vn.result-data-sample.',
-  result_data_index: 'vn.result-data.',
-  pbench_server: 'http://test_domain.com/api/v1',
+  elasticsearch: 'http://elasticsearch.intlab.perf-infra.lab.eng.rdu2.redhat.com:80',
+  results: 'http://pbench.perf.lab.eng.bos.redhat.com:8901',
+  graphql: 'http://graphql.perf.lab.eng.bos.redhat.com:9466/',
+  run_index: 'v6.run.',
+  result_index: 'v5.result-data.',
+  prefix: 'staging-pbench.',
+  pbench_server: 'http://pbench.perf.lab.eng.bos.redhat.com:8901/api/v1',
```

Comment:
This also shouldn't go in the repo.

Reply:
Right, these were changes as a part of the JAM session, will remove